packages/gestalt/src/TableRowExpandable.js

```diff
@@ -59,6 +59,7 @@ export default function TableRowExpandable({
 }: Props): Node {
   const [expanded, setExpanded] = useState(false);
   const cs = hoverStyle === 'gray' ? cx(styles.hoverShadeGray) : null;
+
```

Comment:
This change should be reverted as it's irrelevant to the PR.

Reply:
DONE